Mastodon/Scene/SuggestionAccount/TableViewCell/SuggestionAccountTableViewCell+ViewModel.swift

```diff
@@ -138,4 +74,62 @@ extension SuggestionAccountTableViewCell {
         .assign(to: \.isPending, on: viewModel)
         .store(in: &disposeBag)
     }
+
+    class ViewModel {
+        var disposeBag = Set<AnyCancellable>()
```

Comment:
Can this be made `private`?
```suggestion
        private var disposeBag = Set<AnyCancellable>()
```